lib/AssemblyUtil/FastaToAssembly.py

```diff
@@ -473,6 +482,12 @@ def _validate_mass_params(self, params):
                 raise ValueError(f"Missing {_ASSEMBLY_NAME} field in {_INPUTS} entry #{i}")
         self._get_int(params.get(_MCL), f"If provided, {_MCL}", minimum=2)
 
+        # validate object_metadata is a mapping if provided
+        obj_metas = [i.get(_OBJ_META) for i in inputs]
+        for obj_meta in obj_metas:
+            if obj_meta and type(obj_meta) != dict:
+                raise ValueError(f"{_OBJ_META} must be a mapping if provided")
```

Comment:
Should give the index in the list to the user so it's easier to debug errors

Reply:
👍

Reply:
:+1: